presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeIcebergTpchQueries.java

```diff
@@ -32,7 +32,7 @@ public abstract class AbstractTestNativeIcebergTpchQueries
     protected void createTables()
     {
         QueryRunner queryRunner = (QueryRunner) getExpectedQueryRunner();
-        createLineitemStandard(queryRunner);
+        createLineitemStandard(queryRunner, true);
```

Comment:
The casting should depend on storageFormat as some types like DWRF don't support DATE, but Parquet does. Can you check ?